cloudinit/version.py

```diff
@@ -4,8 +4,15 @@
 #
 # This file is part of cloud-init. See LICENSE file for license information.
 
-__VERSION__ = "25.3"
-_PACKAGED_VERSION = "@@PACKAGED_VERSION@@"
+try:
+    from cloudinit import meson_versions  # type: ignore
```

Comment:
meson build/install will create this module from `meson_versions.py.in` as part of a packaging downstream branch. In it's absence, we'll want to leave a breadcrumb `MISSING_MESON_BUILD_ARTIFACT` which will show up in any packaged downstream which neglects to run meson build for packaging.